javalib/src/main/scala/java/util/concurrent/ConcurrentSkipListSet.scala

```diff
@@ -0,0 +1,423 @@
+package java.util.concurrent
+
+import java.util._
+import java.lang.Comparable
+import scala.collection.mutable
+import scala.math.Ordering
+
+import scala.collection.JavaConversions._
+
+class ConcurrentSkipListSet[E] protected (ordering: Ordering[E], provided: Option[Comparator[E]])
```

Comment:
do not really agree on this, but do not really care, ok

you are right about the type!